turbinia/processors/mount_local.py

```diff
@@ -216,6 +216,69 @@ def PreprocessLosetup(
   return losetup_device
 
 
+def PreprocessMountEwfDisk(ewf_path):
+  """ Locally mounts a EWF disk image.
+
+  Args:
+    ewf_path (str): The path to the EWF image to mount.
+
+  Raises:
+    TurbiniaException: If the mount command failed to run.
+
+  Returns:
+    str: The path to the mounted filesystem.
+  """
+
+  config.LoadConfig()
+  block_prefix = config.MOUNT_DIR_PREFIX
+
+  if not os.path.exists(ewf_path):
+    raise TurbiniaException(
+        'Could not mount partition {0:s}, the path does not exist'.format(
+            ewf_path))
+
+  # Checks if the mount path is a directory
+  if os.path.exists(block_prefix) and not os.path.isdir(block_prefix):
+    raise TurbiniaException(
+        'Mount dir {0:s} exists, but is not a directory'.format(block_prefix))
+
+  # Checks if the mount path does not exist; if not, create the directory
+  if not os.path.exists(block_prefix):
+    log.info('Creating local mount parent directory {0:s}'.format(block_prefix))
+    try:
+      os.makedirs(block_prefix)
+    except OSError as e:
+      raise TurbiniaException(
+          'Could not create mount directory {0:s}: {1!s}'.format(
+              block_prefix, e))
+
+  # Creates a temporary directory for the mount path
+  block_path = tempfile.mkdtemp(prefix='turbinia', dir=block_prefix)
```

Comment:
See my comment in the pre-processor, but maybe we can rename `block_path` to something like `ewf_mount_path` just to be clearer on what this is here?

Reply:
Done